src/ResourceManager/Common/Commands.Common.Strategies/ICmdlet.cs

```diff
@@ -0,0 +1,34 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.Commands.Common.Strategies
+{
+    public interface ICmdlet
+    {
+        void WriteVerbose(string message);
```

Comment:
WriteVerbose, WriteObject, and WriteProgress are already in ICommandRuntime.  Not sure why we need this - seems like just for Parameters.

Also, please provide documentation comemnts on the interface and properties.